web_hide_db_manager_link/README.md

```diff
@@ -0,0 +1,11 @@
+# Hide link to database manager in login screen
```

Comment:
Rename as README.rst and use OCA's template (https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)